README.md

```diff
@@ -129,6 +129,22 @@ Upon hitting `"/marketing/reputation"`,
 `marketingTarget#show` and `reputationTarget#show`
 will be called in that order, and both will be passed the options object.
 
+Finally, route handlers may also be specified as plain functions:
+
+```javascript
+Aviator.setRoutes({
+  '/users': {
+    target: UsersTarget,
+    '/*': 'beforeAll',
+    '/': function () {
+      // Handle the route here
+    }
+  }
+});
+```
+
+Upon hitting `"/users/"`, `usersTarget#beforeAll` and the anonymous function
+above will be called in that order.
```

Comment:
Perhaps add a note that using the targets are encouraged over anonymous functions?